CI have 2 failures

1. `Error: buildx failed with: ERROR: failed to solve: process "/bin/sh -c cargo build --release"`
2. `error[E0599]: no variant or associated item named `Bfloat16` found for enum `TensorElementType` in the current scope`

_Originally posted by @jeffin07 in https://github.com/kornia/kornia-rs/issues/160#issuecomment-2432985412_

--- a/examples/onnx/Cargo.toml
+++ b/examples/onnx/Cargo.toml
@@ -15,7 +15,8 @@ version.workspace = true
 [dependencies]
 clap = { version = "4.5.9", features = ["derive"] }
 kornia.workspace = true
-ort = { version = "2.0.0-rc.4", features = [
+ort = { version = "2.0.0-rc.8", features = [
     "load-dynamic",
+    "half",
 ], default-features = false }
 rerun = "0.18"

--- a/examples/onnx/Cargo.toml
+++ b/examples/onnx/Cargo.toml
@@ -15,6 +15,7 @@ version.workspace = true
 [dependencies]
 clap = { version = "4.5.9", features = ["derive"] }
 kornia.workspace = true
+ort-sys = { version = "2.0.0-rc.8" }
 ort = { version = "2.0.0-rc.8", features = [
     "load-dynamic",
     "half",

